botbowl/core/pathfinding/python_pathfinding.py

```diff
@@ -309,6 +315,9 @@ def _expand(self, node: Node, target=None):
         if self.carries_ball and node.position.x == self.endzone_x:
             return
 
+        if (not self.carries_ball) and node.position == self.game.get_ball_position():
```

Comment:
The ball can be in the air, e.g. during Blitz or Quick Snap! We should make a check for that.

Reply:
Done.